japanese/newyear/newyear

```diff
@@ -0,0 +1,2 @@
+あけましておめでとうございます。今年もよろしくお願いします。
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.